libraryBuilds/gson/build.gradle

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2015 the authors of j2objc-common-libs-e2e-test
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        // This is the build output of the plugin itself.
+        classpath fileTree(dir: '../../j2objc-gradle/build/libs', include: ['*.jar'])
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'com.github.j2objccontrib.j2objcgradle'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    j2objcTranslation 'com.google.code.gson:gson:2.3.1:sources'
+}
+
+j2objcConfig {
+    // Multiple Java filenames can indeed be identical.
+    filenameCollisionCheck false
```

Comment:
This shouldn't be done by default... if it is really needed, I was expecting a comment like:

    // Suppress existing filename collision:
    // src/main/abc/File1.java
    // src/main/xyz/File1.java

Are you using `no-package-directories` for the translation? If there is a real collision going on, then I think that prefixes.properties can help fix that.